__tests__/app.test.js

```diff
@@ -41,10 +41,28 @@ describe("GET/api", () => {
     });
 })
 
-
-
-
-
+describe("GET/api/articles", () => {
+    test("200: gets all articles, with the body property removed, a commment count added and sorted by date in descending order", () => {
```

Comment:
You've mentioned that the body property should be removed, can you assert in the test that this is the case

Note: this test block is doing a LOT of work. in TDD we really want to contain behaviours in their own tests. Really this looks like four tests:
1. gets all articles
2. body property isn't there
3. comment count is there
4. it is sorted by date in desc order default

Reply:
Noted, will replace the keys.length check with a test that body is not a key on the object.  Will also make a note to split these out into separate tests at some point and will do that going forward